### ISSUE TITLE
undefined method `losses=' 

### ISSUE THREAD

**Issue by Theminijohn** [mikamai/ruby-lol#35]
I'm trying to get the League information of a summoner:

```ruby
client = Lol::Client.new Settings.rito.api_key, {region: self.server}

client.league.get(20964624)
```

which gives me an output :

```console
[3] pry(main)> client.league.get(20964624)
NoMethodError: undefined method `losses=' for #<Lol::LeagueEntry:0x007fd6edda9578>
from /Users/theminijohn/.rvm/rubies/ruby-2.2.0/lib/ruby/gems/2.2.0/gems/ruby-lol-0.9.19/lib/lol/model.rb:15:in `block in initialize'
```

I tried getting the League Entry, but couldn't get it working, how can I access  the LeagueEntry and make a request?

What am I missing?

**Comment by intinig** [comment]
It's a problem with a yet not implemented change on the API.

I'll allocate some time to fix this tomorrow and push a new version out.

**Comment by Theminijohn** [comment]
Would be awesome :) Thanks a lot !

**Comment by intinig** [comment]
Hello,

the just released gem 0.9.19.1 still doesn't fix this problem. Hopefully it will be fixed today.

**Comment by Theminijohn** [comment]
Thanks for the Heads up !
